consensus/poet/common/sawtooth_poet_common/sgx_structs/_sgx_attributes.py

```diff
@@ -0,0 +1,93 @@
+#
+# INTEL CONFIDENTIAL
```

Comment:
This is not the same license we are using in the rest of the project - needs to be the Apache 2.0 license